xbmc/interfaces/python/PythonInvoker.cpp

```diff
@@ -101,6 +101,12 @@ static std::vector<std::vector<char>> storeArgumentsCCompatible(std::vector<std:
   std::vector<std::vector<char>> output;
   std::transform(input.begin(), input.end(), std::back_inserter(output),
                 [](std::string const & i) { return std::vector<char>(i.c_str(), i.c_str() + i.length() + 1); });
+
+  if(output.empty())
+  {
+    output.push_back(std::vector<char>(1u, '\0'));
+  }
```

Comment:
Style nitpick - can you make this

```c++
if (output.empty())
  output.push_back(std::vector<char>(1u, '\0'));
```

It looks like you're on a roll with fixes, so future ones should follow our coding style or the bike shedding police will speak up :)